views/apartments/new.erb

```diff
@@ -1,8 +1,8 @@
 <form method="post" action="/apartments">
-  <input type="text" placeholder="Address">
-  <input type="text" placeholder="Monthly Rent">
-  <input type="text" placeholder="Square Feet">
-  <input type="text" placeholder="Number Beds">
-  <input type="text" placeholder="Number Baths">
+  <input type="text" placeholder="Address" name="address">
+  <input type="text" placeholder="Monthly Rent" name="monthly_rent">
+  <input type="text" placeholder="Square Feet" name="sqft">
+  <input type="text" placeholder="Number of Beds" name="num_beds">
+  <input type="text" placeholder="Number of Baths" name="num_baths">
```

Comment:
Re my earlier comment, here's where you'd adjust the `name` attributes to be 'namespaced'.